python/lsst/ap/verify/testPipeline.py

```diff
@@ -463,17 +505,28 @@ def run(self, diaSourceCat, diffIm, band, ccdVisitId, funcs=None):
                       )
 
 
+class MockDiaPipelineConfig(DiaPipelineConfig):
+
+    def setDefaults(self):
+        super().setDefaults()
+        # Avoid delegating to lsst.obs.base.Instrument specialization for the
+        # data ID packing algorithm to use, since test code often does not use a
+        # real Instrument in its data IDs.
+        self.idGenerator.packer.name = "observation"
```

Comment:
What exactly is this ID generator, that _every_ task needs to use it? It seems like the sort of thing that `DiaPipelineTask` wouldn't need to bother with.

Reply:
It's a way to pack data IDs (`{visit, detector}` ones, in this case) and maybe release IDs into single integers.  It's going everywhere `lsst.obs.base.ExposureIdInfo` was once used.